e2e/__tests__/__snapshots__/failures.test.js.snap

```diff
@@ -360,8 +360,9 @@ exports[`works with async failures 1`] = `
 
     expect(received).rejects.toEqual()
 
-    Expected received Promise to reject, instead it resolved to value
-      {\\"foo\\": \\"bar\\"}
+    Received promise resolved instead of rejected
+
+    Received promise resolved to value: {\\"foo\\": \\"bar\\"}
```

Comment:
Updated picture following suggestion, except for dot (because most messages don’t have it).

<img width="1000" alt="instead-1-resolves-false" src="https://user-images.githubusercontent.com/11862657/50702106-f90d3880-101d-11e9-8e4d-97ef1dc8a636.png">